package.json

```diff
@@ -47,15 +47,18 @@
     "dependents-filter:semistandard": "npm run --silent dependents-filter-one -- -i dependents-data/source/semistandard.ndjson -o dependents-data/semistandard-filtered.ndjson --min-downloads=1000 --target-version='>=16.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/semistandard.json)",
     "dependents-filter:standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/standard.ndjson -o dependents-data/standard-filtered.ndjson --min-downloads=75000 --target-version='>=17.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/standard.json)",
     "dependents-filter:ts-standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/ts-standard.ndjson -o dependents-data/ts-standard-filtered.ndjson --min-downloads=1000 --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/ts-standard.json)",
-    "dependents-filter": "run-p dependents-filter:*",
-    "dependents-refresh-one": "cd dependents-data/source/ && list-dependents refresh --field engines -n",
+    "dependents-filter-simplify": "for PROJECT in $(jq -r 'join(\" \")' dependents-data/projects.json); do jq -s '[.[].repositoryUrl | sub(\"https://github.com/\";\"\";\"i\")] | unique | sort' \"dependents-data/$PROJECT-filtered.ndjson\" > \"dependents-data/$PROJECT.json\"; done",
+    "dependents-filter": "run-p dependents-filter:* && run-s dependents-filter-simplify",
+    "dependents-refresh-one": "cd dependents-data/source/ && list-dependents refresh --precision 3 --field engines -n",
     "dependents-refresh": "run-s --print-label \"dependents-refresh-one -- {%}\" -- $(jq -r 'join(\" \")' dependents-data/projects.json)",
-    "dependents-update-one": "cd dependents-data/source/ && list-dependents list --field engines -n",
+    "dependents-update-one": "cd dependents-data/source/ && list-dependents list --precision 3 --field engines -n",
```

Comment:
`--precision 3` is a new feature in [`list-dependents-cli`](https://github.com/voxpelli/list-dependents-cli) (see https://github.com/voxpelli/list-dependents-cli/pull/30) that I dogfooded here. It rounds the download numbers so that the first three digits are always zeroes. That way the source dependent data will not get modified every single time a download count goes up or down but only if it moves from one bucket to another.